### DISCUSSION TITLE
Greenlet and gevent versions to run asreview lab

### DISCUSSION THREAD

**Discussion by ljcolson** [asreview/asreview#1273] [Troubleshooting (report errors in the "Issues" tracker)]
I was getting the following error when running the command "asreview lab" in the terminal:

`importlib/_bootstrap.py:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject return f(*args, **kwds)`
`Segmentation fault: 11`

I was using 
greenlet version: 1.1.1
gevent version 1.4.0

I fixed this error by upgrading the python modules greenlet and gevent using the following commands:

"pip install --upgrade gevent" and "pip install --upgrade greenlet"

Now running:
greenlet version: 2.0.1
gevent version: 22.10.2

These versions work for me. 

Hope this helps others who are troubleshooting.

**Comment by J535D165** [comment]
Thanks for reporting and the fix. 

I'm wondering what the previous version was. Gevent is indeed a dep of asreview, we might want to set a min version.

**Reply by jteijema** [reply]
Version `20.4.0` runs ASReview for me, while `1.5.0` gives these errors. I should note that those are [consecutive versions](https://pypi.org/project/gevent/#history).

**Reply by J535D165** [reply]
Is anyone interested in doing a PR?

**Reply by jteijema** [reply]
yes, see #1276!
